mapping/field.go

```diff
@@ -26,8 +26,9 @@ import (
 
 // control the default behavior for dynamic fields (those not explicitly mapped)
 var (
-	IndexDynamic = true
-	StoreDynamic = true
+	IndexDynamic     = true
+	StoreDynamic     = true
+	DocValuesDynamic = true // may be default disabled?
```

Comment:
we should be consistent with the naming, you chose IncludeDocValues earlier, we have to stick with it, or change them all to just "DocValues"